lib/auth.js

```diff
@@ -16,6 +16,19 @@ exports.AUTH_STRATEGY = 'dogfood';
 exports.AUTH_SCHEME = 'bearer';
 
 exports.SCOPE_CLIENT_MANAGEMENT = 'oauth';
+exports.SCOPE_TOKEN_MANAGEMENT = 'oauth:tokens';
+
+function canManageClients(scopes) {
+  for (var i = 0, len = scopes.length; i < len; i++) {
+    var scope = scopes[i];
+    if (scope === exports.SCOPE_CLIENT_MANAGEMENT ||
+      scope.lastIndexOf(exports.SCOPE_CLIENT_MANAGEMENT + ':', 0) === 0) {
+
+      return true;
+    }
+  }
+  return false;
+}
```

Comment:
@seanmonstar's refresh-tokens PR has a nice refactor that will simplify this; https://github.com/mozilla/fxa-oauth-server/pull/260/files#r31217134